staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -838,6 +838,11 @@ func describePod(pod *corev1.Pod, events *corev1.EventList) (string, error) {
 			w.Write(LEVEL_0, "NominatedNodeName:\t%s\n", pod.Status.NominatedNodeName)
 		}
 
+		if pod.Spec.Resources != nil {
+			w.Write(LEVEL_0, "Resources:\n", "")
```

Comment:
nit: I think the empty string for the third argument is also unnecessary.

```suggestion
			w.Write(LEVEL_0, "Resources:\n")
```